### ISSUE TITLE
GazePrompt: Enhancing Low Vision People's Reading Experience with Gaze-Aware Augmentations

### ISSUE THREAD

**Issue by Masa630** [SeitaKayukawa/Blind_Accessibility_Papers#615]
### Links
- Paper: https://arxiv.org/pdf/2402.12772

### Abstract
Reading is a challenging task for low vision people. While conventional low vision aids (e.g., magnification) offer certain support, they cannot fully address the difficulties faced by low vision users, such as locating the next line and distinguishing similar words. To fill this gap, we present GazePrompt, a gaze-aware reading aid that provides timely and targeted visual and audio augmentations based on users' gaze behaviors. GazePrompt includes two key features: (1) a Line-Switching support that highlights the line a reader intends to read; and (2) a Difficult-Word support that magnifies or reads aloud a word that the reader hesitates with. Through a study with 13 low vision participants who performed well-controlled reading-aloud tasks with and without GazePrompt, we found that GazePrompt significantly reduced participants' line switching time, reduced word recognition errors, and improved their subjective reading experiences. A follow-up silent-reading study showed that GazePrompt can enhance users' concentration and perceived comprehension of the reading contents. We further derive design considerations for future gaze-based low vision aids.

<img width="1013" alt="スクリーンショット 2024-06-13 23 25 34" src="https://github.com/SeitaKayukawa/Blind_Accessibility_Papers/assets/127022571/bf3849df-ab7c-462e-a201-c88dde6cd1ae">

低視力者が文章を読むことを補助するシステム
二つのサポート機能を持つ
・Line switching Support: Line Highlight / Arrow
・Difficult-Word Support: Word Magnifier / Text-to-Speech

_“Despite early successes in collecting low vision people’s gaze data, no research has explored how to design effective gaze-aware technologies to enhance their visual experiences.”_ であるため本研究には貢献がある

13名+12名の低視力者を対象にユーザ実験
・ユーザ実験は　1.文章を読み上げる実験　2.文章を静かに読んで理解する実験 の二つを行った
・全ての参加者が提案手法を高く評価した
・Line switching SupportはArrowよりもLine Highlightを好んだ
・Difficult-Word SupportはMagnifierよりもText-to-Speechを好んだ
